local/sites/test-site/env.yml

```diff
@@ -18,25 +18,22 @@ agent:
 metadata:
   name: detroit-preprod
   kind: pre-prod
-
+  etcd_initial_cluster: etcd-c1=http://10.10.10.10:2380,etcd-c2=http://10.10.10.11:2380,etcd-c3=http://10.10.10.12:2380
```

Comment:
We can define the `etcd_ignition_cluster` here too. I think its a lot cooler to show the loop enumeration in `local/assets/ignition/etcd.yml`, if we can keep that for one of them that will work. It requires each host to define that metadata